website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,304 @@
+   ```
+6. Karpenter continues to watch the node, waiting until the node becomes ready, has all its startup taints removed, 
+   and has all requested resources registered on the node. 
+   Example log:
+    ```
+    {"level":"INFO","time":"2024-06-22T02:24:52.642Z","logger":"controller","message":
+        "initialized nodeclaim","commit":"490ef94","controller":
+        "nodeclaim.lifecycle","controllerGroup":"karpenter.sh","controllerKind":
+        "NodeClaim","NodeClaim":{"name":"default-sfpsl"},"namespace":
+        "","name":"default-sfpsl","reconcileID":
+        "7e7a671d-887f-428d-bd79-ddf603290f0a",
+        "provider-id":"aws:///us-west-2b/i-01234567adb205c7e",
+        "Node":{"name":"ip-xxx-xxx-xx-xxx.us-west-1.compute.internal"},
+        "allocatable":{"cpu":"7910m","ephemeral-storage":"18242267924",
+        "hugepages-2Mi":"0","memory":"14320468Ki","pods":"58"}}
+    ```
+
+## NodeClaim drift and disruption
+
+NodeClaims are immutable and cannot be modified, but are used to compare to changes made to the NodePool. When NodePools are modified, differing in spec from a created NodeClaim, Karpenter will mark the NodeClaim as drifted.
+
+For details on Karpenter disruption, see [Disruption]({{< ref "./disruption" >}}).
+
+## NodeClaim example
+The following is an example of a NodeClaim. Keep in mind that you cannot modify a NodeClaim.
+To see the contents of a NodeClaim, get the name of your NodeClaim, then run `kubectl describe` to see its contents:
+
+```
+kubectl get nodeclaim
+NAME            TYPE               ZONE         NODE                                           READY   AGE
+default-m6pzn   c7i-flex.2xlarge   us-west-1a   ip-xxx-xxx-xx-xxx.us-west-1.compute.internal   True    7m50s
+
+kubectl describe nodeclaim default-m6pzn
+```
+Starting at the bottom of this example, here are some highlights of what the NodeClaim contains:
+
+* The Node Name (ip-xxx-xxx-xx-xxx.us-west-1.compute.internal) and Provider ID (aws:///us-west-1a/i-xxxxxxxxxxxxxxxxx) identify the instance that is fulfilling this NodeClaim.
+* Image ID (ami-0ccbbed159cce4e37) represents the operating system image running on the node.
+* Status shows the resources that are available on the node (CPU, memory, and so on) as well as the conditions associated with the node. The conditions show the status of the node, including whether the node is launched, registered, and healthy. This is particularly useful if Pods are not deploying to the node and you want to determine the cause.
+* The Spec sections show the values of different NodeClaim objects. For example, you can see the type of operating system running (linux), the instance type and category, the NodePool that was used for the NodeClaim, the node's architecture (amd64), and resources such as CPU and number of Pods currently running on the node.
+* Metadata show information about how the NodeClaim was created.
```

Comment:
We can probably drop this line, it's the standard metadata in any kubernetes resource. If we want to keep it, we should probably go into a little more detail (e.g. the finalizer and owner ref to the NodePool).